crates/biome_css_analyze/tests/specs/nursery/noInvalidPositionAtImportRule/validCharsetMultipleImport.css.snap

```diff
@@ -0,0 +1,11 @@
+---
+source: crates/biome_css_analyze/tests/spec_tests.rs
+expression: validCharsetMultipleImport.css
+---
+# Input
+```css
+@CHARSET 'utf-8';
```

Comment:
major

This is invalid.
i.e. `@charset` string must be double-quoted.

Reply:
fix: https://github.com/biomejs/biome/pull/2717/commits/27b6ab54935f88be03e5d1129df27cc8bc07f135